lib/transaction/transaction.js

```diff
@@ -119,8 +114,8 @@ Object.defineProperty(Transaction.prototype, 'id', hashProperty);
 
 
 Transaction.prototype.getValueOfBackwardTransfers = function() {
-  var amount = 0;
-  if (this.version == -5){
+  let amount = 0;
+  if (this.version === -5){
```

Comment:
Please add an enum which describes the different kind of transactions based on the version.

Reply:
Yes, but I need your help for this task. I need to know what the different numbers mean